tests/utils/azure_users_test.py

```diff
@@ -176,3 +176,45 @@ def test_get_groups_and_users_called_once_by_instance(
     azure_connection.get_groups_by_user_id(user_id)
 
     get_groups_and_users_mock.assert_called_once()
+
+
+@patch('utils.azure_users.AzureConnection.get_msal_client', Mock())
+@patch('utils.azure_users.AzureConnection.get_token', Mock())
+@patch('utils.azure_users.AzureConnection.get_user')
+@patch('utils.azure_users.AzureConnection.get_group_id_by_group_name')
+@patch('requests.post')
+def test_add_user_to_group(
+    post_mock, get_group_id_by_group_name_mock, get_user_mock
+):
+    get_group_id_by_group_name_mock.return_value = 'dummy_group'
+
+    response_mock = Mock()
+    response_mock.status_code = 204
+    post_mock.return_value = response_mock
+
+    azure_connection = AzureConnection()
+    azure_connection.add_user_to_group('dummy_user_id', 'dummy_group')
+
+    get_group_id_by_group_name_mock.assert_called_once()
```

Comment:
Do you think you can send an assert with the mock of a user in the test that is changing the group? It is optional because I see that if you are testing the methods.